src/wp-admin/includes/update-core.php

```diff
@@ -18,6 +18,8 @@
  */
 global $_old_files;
 
+require_once ABSPATH . 'wp-includes/Requests/Exception.php';
```

Comment:
Is this the old missing "from" version file in memory that might be needed during the upgrade process? It's the one directly involved in the ticket. But have we confirmed this is the only one that is needed? 

For example, there are other types of exceptions in the Requests library. See the `Exception` folder on [5.5](https://github.com/WordPress/wordpress-develop/tree/5.5/src/wp-includes/Requests/Exception), [5.6](https://github.com/WordPress/wordpress-develop/tree/5.6/src/wp-includes/Requests/Exception), [5.7](https://github.com/WordPress/wordpress-develop/tree/5.7/src/wp-includes/Requests/Exception) and [5.8](https://github.com/WordPress/wordpress-develop/tree/5.8/src/wp-includes/Requests/Exception).

May want to explore what files in the HTTP and Requests library are used and then load them here.

Reply:
> > This line will be needed for every branch of WordPress that uses the Requests library (both past and future branches).
> 
> @costdev The upgrader uses the "to" version `wp-admin/includes/update-core.php` file, meaning only 5.9 needs this code. For older WordPress versions upgrading to 5.9, this PR will ensure that missing "from" version exception file is loaded into memory from the older WordPress version.
> 
> Why then does this need to be backported to all the older branches?

"to" is only 5.9 when upgrading. When rolling back, "to" becomes the earlier version, and the fatal is thrown on missing `WpOrg\Requests\Exception`.

Reply:
Another idea is to load all of the Requests files into memory. That way all are available for any edge case or exception that might happen.